Description: Various build tweaks
 * Specifies the source/target level
 * Fixes the version in the title of the Javadoc
 * Removes the private fields/methods from the Javadoc
 * Disables the tests (missing dependencies)
Forwarded: not-needed
--- a/build.xml	2009-09-13 12:20:12.177029733 +0200
+++ b/build.xml	2009-09-13 12:20:44.817007573 +0200
@@ -33,7 +33,7 @@
   <target name="compile" description="o Compile the code" depends="get-deps">
     <mkdir dir="${classesdir}">
     </mkdir>
-    <javac destdir="${classesdir}" deprecation="true" debug="true" optimize="false" excludes="**/package.html">
+    <javac destdir="${classesdir}" deprecation="true" debug="true" optimize="false" excludes="**/package.html" target="1.3" source="1.3">
       <src>
         <pathelement location="${basedir}/src/java">
         </pathelement>
@@ -52,7 +52,7 @@
       </fileset>
     </copy>
   </target>
-  <target name="jar" description="o Create the jar" depends="compile,test">
+  <target name="jar" description="o Create the jar" depends="compile">
     <jar jarfile="${defaulttargetdir}/${final.name}.jar" excludes="**/package.html" basedir="${classesdir}">
     </jar>
   </target>
@@ -144,9 +144,9 @@
     </tstamp>
     <property name="copyright" value="Copyright &amp;copy;  The Apache Software Foundation. All Rights Reserved.">
     </property>
-    <property name="title" value="JXPath 1.2 API">
+    <property name="title" value="JXPath 1.3 API">
     </property>
-    <javadoc use="true" private="true" destdir="${javadocdir}" author="true" version="true" sourcepath="src/java" packagenames="*">
+    <javadoc destdir="${javadocdir}" author="true" version="true" sourcepath="src/java" packagenames="*">
       <classpath>
         <fileset dir="${libdir}">
           <include name="*.jar">
